infrastructure/server/variables_provided.tf

```diff
@@ -115,8 +115,9 @@ variable "front_end_redirect_path" {
   type = string
 }
 
-variable "local_frontend_redirect_path" {
-  description = "Path to local FAM front-end (for redirect URI), only for dev"
+variable "api_gateway_stage_name" {
+  description = "Stage name for the REST API in API Gateway (appears in URI)"
```

Comment:
We should be able to control the stage name (e.g. have it consistent per environment) so not sure why its a variable, but this is just a question, the PR can go ahead despite this.

Reply:
If we want this to be different for different environments, we can override in the terragrunt files (per environment)!